Cargo.toml

```diff
@@ -24,6 +24,7 @@ default = ["std"]
 integer128 = []
 std = ["serde/std"]
 indexmap = ["std", "dep:indexmap"]
+span-substring-test = ["unicode-segmentation"]
```

Comment:
Can you add a comment here that this feature is internal and should not be used in user code (maybe even prepend "internal-" to the name?)